flake.nix

```diff
@@ -393,16 +389,16 @@
 
             # TODO: inherit and provide script to run all stuff
             # devnet-container-xcvm
+            # NOTE: The devcontainer is currently broken. Please use the developers devShell
             devcontainer = dockerTools.buildLayeredImage {
               name = "composable-devcontainer";
               fromImage = devcontainer-base-image;
               # be very carefull with this, so this must be version compatible with base and what vscode will inject
               contents = [
                 # ISSUE: for some reason stable overrides nighly, need to set different order somehow
                 #rust-stable
-                rust-nightly-dev
+                rust-nightly
```

Comment:
not sure, you removed RA either via packages and also via overlay. so yeah. it definetely broken now.

Reply:
please try yourself remote without these changes.

Reply:
RA is shipped via the rust overlay

Reply:
@dzmitry-lahoda please check out line 103, rust-analyzer is now managed by the rust overlay so that the version exactly matches the compiler version

Reply:
i see. not sure if RA is needed to build rust-nightly .  it is possible to make rust-nightly. and dev is rust-nightly.mkOverride(self: { extensions = self.exteions ++ [ ra] ))

Reply:
We can do that, nice suggestion. I'll do it now